notebooks/community/model_garden/model_garden_mediapipe_gesture_recognition.ipynb

```diff
@@ -0,0 +1,557 @@
+{
```

Comment:
<p>I like to suggest using the official notebook template version for setting PROJECT_ID, REGION, and BUCKET_URI</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/2129/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Reply:
Done (everywhere)